### PR TITLE
chore(quickstart): add jaxb-impl dependency and help with using Tasklist and Modeler UI

### PR DESCRIPTION
**Pull request by alexcorremans**
- require dependency jaxb-impl for Java 9 and above
- use the create space tool before adding the user task
- add a task filter in Tasklist
- explain how to configure input column in DMN table

related to CAM-14436

### CODE REVIEW

File: get-started/content/quick-start/install.md [hunk 1]
```diff
@@ -47,6 +47,10 @@ First, download a distribution of the Camunda Platform. You can choose from diff
 
 After downloading the distribution, unpack it inside a directory of your choice.
 
+{{< note class="info" title="Additional requirement for Java 9 or above" >}}
+If you are using Java 9 or above, you additionally need a JAXB implementation. Download the [jaxb-impl dependency](https://repo1.maven.org/maven2/com/sun/xml/bind/jaxb-impl/2.3.3/jaxb-impl-2.3.3.jar) and copy it to the following path of the `camunda-bpm-run` application:
+`/camunda-bpm-run-7.x.x/configuration/userlib` {{< /note >}}
```
Comment (marstamm) [inline get-started/content/quick-start/install.md:52]:
```suggestion
`/camunda-bpm-run-7.x.x/configuration/userlib`
{{< /note >}}
```

**Review by marstamm — COMMENTED** [review]
Looks okay. Added a few styling suggestions

**Review by ThorbenLindhauer — CHANGES_REQUESTED** [review]
Thanks for your improvements. I'm questioning if the jaxb dependency is needed, please see my detailed comment.

File: get-started/content/quick-start/install.md [hunk 2]
```diff
@@ -47,6 +47,11 @@ First, download a distribution of the Camunda Platform. You can choose from diff
 
 After downloading the distribution, unpack it inside a directory of your choice.
 
+{{< note class="info" title="Additional requirement for Java 9 or above" >}}
+If you are using Java 9 or above, you additionally need a JAXB implementation. Download the [jaxb-impl dependency](https://repo1.maven.org/maven2/com/sun/xml/bind/jaxb-impl/2.3.3/jaxb-impl-2.3.3.jar) and copy it to the following path of the `camunda-bpm-run` application:
+`/camunda-bpm-run-7.x.x/configuration/userlib`
+{{< /note >}}
```
[the remark this answers is not in the document]
Comment (alexcorremans) [reply get-started/content/quick-start/install.md:53]: You got me! I was indeed running Java 17. I've now tested both with Java 15 and 7.16.0, and Java 17 with 7.17.0-alpha6, and can confirm both are working. 

I agree the guide should be more explicit on using a supported Java version. I'll work on that now.

**Review by ThorbenLindhauer — APPROVED** [review]

**Comment by ThorbenLindhauer** [comment]
Thanks for the contribution, I merged it via https://github.com/camunda/camunda-docs-static/commit/8a17972ee62ca080cedf747963c607126e1d6581 to master. We will release the changes with the 7.17 release next week.

**Pull request closed without merging**